SU2_PY/SU2/mesh/tools.py

```diff
@@ -29,6 +29,11 @@
 # You should have received a copy of the GNU Lesser General Public
 # License along with SU2. If not, see <http://www.gnu.org/licenses/>.
 
+import sys
+if sys.version_info.major > 2:
+    # In PY3, 'long' and 'int' are unified in 'int' type
+    long = int
```

Comment:
In python 3, long is renamed int and is equivalent to py2 "long".
Here, line long = int is applied only in case of python 3 so normally it changes nothing on python2